skunkbooth/main.py

```diff
@@ -28,6 +29,17 @@
 )
 
 
+def global_shortcuts(event):
+    """Event handler for global shortcuts"""
+    ctrlQCode = 17
```

Comment:
Where possible, we probably want to import and use the named variable in the enumeration from asciimatics instead of set keycodes as rigid numbers because asciimatics might change the keycode internally

Reply:
Commit dbc3f42 should have fixed this. I swapped  the hard-coded values for the Screen.ctrl() function